app/tests/integration/nimbus/test_desktop_client_integrations.py

```diff
@@ -279,18 +261,9 @@ def test_check_telemetry_sticky_targeting(
         "description": "reference branch",
         "name": "Branch 1",
         "ratio": 100,
-        "featureEnabled": True,
         "featureValue": "{}",
     }
-    experiment_default_data["treatmentBranches"] = [
-        {
-            "description": "treatment branch",
-            "name": "Branch 2",
-            "ratio": 0,
-            "featureEnabled": False,
-            "featureValue": "",
-        }
-    ]
+    experiment_default_data["treatmentBranches"] = []
```

Comment:
this thing 🆙

Reply:
Oh the tests don't use those branches so there's no point in them being declared at all.